ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/templates/Template3.kt

```diff
@@ -88,6 +91,11 @@ private fun ColumnScope.PortraitContent(state: PaywallState.Loaded, viewModel: P
             horizontalAlignment = Alignment.CenterHorizontally,
             verticalArrangement = Arrangement.spacedBy(UIConstant.defaultVerticalSpacing, Alignment.Top),
         ) {
+            Spacer(
```

Comment:
Since we repeat this spacer in a few places, would it be worth to extract to a different composable?

Reply:
good idea yes

Reply:
Done in https://github.com/RevenueCat/purchases-android/pull/1613/commits/bff274f46184dd3aa416b77be22a2192ca4da8c1